src/victron_mqtt/_victron_topics.py

```diff
@@ -726,4 +734,60 @@
         value_type=ValueType.FLOAT,
         precision=1,
     ),
+    # temperature devices
+    "N/+/temperature/+/Temperature": TopicDescriptor(
+        message_type=MetricKind.SENSOR,
+        short_id="temperature",
+        name="Temperature",
+        unit_of_measurement="°C",
+        metric_type=MetricType.TEMPERATURE,
+        metric_nature=MetricNature.INSTANTANEOUS,
+        device_type=DeviceType.TEMPERATURE,
+        value_type=ValueType.INT,
```

Comment:
This seems to work as is and displays like this:

<img width="334" height="112" alt="Screenshot 2025-07-16 at 16 01 20" src="https://github.com/user-attachments/assets/57c7f91b-a123-4c17-8ca0-52965d459b1e" />

Which matches what I see on the Cerbo GX:

<img width="475" height="879" alt="Screenshot 2025-07-16 at 16 01 50" src="https://github.com/user-attachments/assets/6d554077-bd7b-4ab4-ae36-d8a9f6de9e37" />

I did previously see a decimal value here though. Would that still come through now `value_type` is `INT`? Does it make any sense to set a `precision` when `value_type=VALUE_TYPE.INT`?